src/GovUk.Education.ExploreEducationStatistics.Admin/Services/Interfaces/IPublicationRepository.cs

```diff
@@ -8,7 +8,9 @@ namespace GovUk.Education.ExploreEducationStatistics.Admin.Services.Interfaces
 {
     public interface IPublicationRepository
     {
-        Task<List<Publication>> GetAllPublicationsForTopic(Guid topicId);
+        Task<List<Publication>> GetPublicationsForTopic(Guid topicId);
+
+        Task<List<Publication>> GetPublicationListForUser(Guid userId, Guid? topicId = null);
```

Comment:
I originally named it that then switched to `List` because there is already a `GetPublicationForUser` and the `List` helped make the two different methods easier to distinguish. All that said, I was thinking this would get pulled up in review - happy to go along with what others think is best.

Reply:
Ben commented saying we're now naming methods that return lists differently, so I've changed this to `ListPublicationByUser`.